test/Serialization/attr-actorindependent.swift

```diff
@@ -0,0 +1,32 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -enable-experimental-concurrency -emit-module -o %t/a.swiftmodule -primary-file %s -module-name main
+// RUN: llvm-bcanalyzer -dump %t/a.swiftmodule | %FileCheck %s
```

Comment:
An addition check you might add is to use `%target-swift-ide-test` to print out `%t/a.swiftmodule` and use FileCheck to ensure that you get `@actorIndependent` and `@actorIndependent(unsafe)` where you expect it. That'll test the round-tripping aspect.